node/cn/backend.go

```diff
@@ -887,10 +880,6 @@ func (s *CN) Start(srvr p2p.Server) error {
 		s.lesServer.Start(srvr)
 	}
 
-	if !s.chainConfig.IsKaiaForkEnabled(s.blockchain.CurrentBlock().Number()) {
-		reward.StakingManagerSubscribe()
```

Comment:
Has this subscription been moved or completely deleted here?

Reply:
Completely deleted because StakingManager does not exist anymore.

Reply:
Yes I deleted it from staking_manager.go

Reply:
Oh wait, I think I should implement a similar logic in kaiax/staking that it does WriteStakingInfo in PostInsertBlock. That was the purpose of handleChainHeadEvent().

Reply:
PTAL [7ebc36f](https://github.com/kaiachain/kaia/pull/133/commits/7ebc36f5eb0d08216d759d6121d128d439283459)

Reply:
`PostInsertBlock` contributeed to its right place :)